testing-suite/staking-v4/get_infos/get_validator_info.py

```diff
@@ -101,39 +69,90 @@ def get_owner(public_validator_key: list[str]) -> str:
     return address
 
 
-def check_if_error_is_present_in_tx(error, tx_hash) -> bool:
-    flag = False
-    error_bytes = string_to_base64(error)
+# using validator/statistics
+def get_keys_state(keys: list) -> list[str]:
+    states = []
+
+    force_reset_validator_statistics()
+    time.sleep(1)
 
-    req = requests.get(f"{DEFAULT_PROXY}/transaction/{tx_hash}?withResults=True")
-    response = req.text
+    response = requests.get(f"{OBSERVER_META}/validator/statistics")
+    response.raise_for_status()
 
-    if error_bytes.decode() in response:
-        flag = True
+    parsed = response.json()
 
-    if error in response:
-        flag = True
+    general_data = parsed.get("data")
+    statistics = general_data.get("statistics")
 
-    return flag
+    for key in keys:
+        if not statistics.get(key) is None:
+            key_data = statistics.get(key)
+            state = key_data.get("validatorStatus")
+            states.append(state)
 
+    return states
 
-def get_total_staked(owner: str):
-    address_in_hex = Address.from_bech32(owner).to_hex()
-    post_body = {
-        "scAddress": VALIDATOR_CONTRACT,
-        "funcName": "getTotalStaked",
-        "args": [address_in_hex]
-    }
 
-    json_structure = json.dumps(post_body)
-    response = requests.post(f"{DEFAULT_PROXY}/vm-values/query", data=json_structure)
+def get_keys_from_validator_auction(QUALIFIED=True) -> list[str]:
+    keys = []
+
+    force_reset_validator_statistics()
+    time.sleep(1)
+
+    response = requests.get(f"{OBSERVER_META}/validator/auction")
     response.raise_for_status()
     parsed = response.json()
 
     general_data = parsed.get("data")
-    tx_response_data = general_data.get("data")
-    total_staked_list = tx_response_data.get("returnData")
-    total_staked = total_staked_list[0]
+    auction_list_data = general_data.get("auctionList")
+
+    for list in auction_list_data:
+        nodes_lists = list.get("nodes")
+        for node_list in nodes_lists:
+            if node_list.get("qualified") == QUALIFIED:
+                keys.append(node_list.get("blsKey"))
+
+    return keys
+
+
+def get_keys_from_validator_statistics(needed_state: str) -> list[str]:
```

Comment:
It is already in get_infos folder. We had before a get_info.py, but I divided it into multiple files based on from where it will take the information : staking, delegation, transactions...etc. I renamed it network_provider